bower.json

```diff
@@ -19,6 +19,7 @@
     "locales": "https://github.com/citation-style-language/locales.git#d2b612f8a6f764cbd66e67238636fac3888a7736",
     "raven-js": "2.1.0",
     "zeroclipboard": "2.1.6",
-    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96"
+    "osf-style": "https://github.com/CenterForOpenScience/osf-style.git#5dcf5997ad2808ffc7c8564f94cde102bf7a4a96",
+    "At.js": "https://github.com/leb2dg/At.js.git#d4c1c6ebce3808a275bac59740dabd9bcf76d96f"
```

Comment:
Using personal forks as dependencies is a bad practice, because it introduces a source of instability for the OSF. What happens when, some day long after you've left COS, you decide to prune your GitHub repos? Even before then, are you going to keep your fork up-to-date with the upstream project?

If a third-party project doesn't meet our requirements, we should:

 1. Fork to the CenterForOpenScience GitHub organization.
 1. Submit a PR from the CenterForOpenScience to the third party, and ask for a new release.
 1. If they don't make a new release in a reasonable amount of time, only then would we use the COS fork as a dependency.
 1. Add a notice at the top of our fork's README documenting why we forked and linking to the PR we made.